lib/elixir/lib/calendar.ex

```diff
@@ -265,6 +265,15 @@ defmodule Calendar do
   """
   @callback valid_time?(hour, minute, second, microsecond) :: boolean
 
+
+  @doc """
+  Parses the string representation for a time returned by `c:time_to_string/4`
+  into a time-tuple.
+  """
+  @callback parse_time(String.t()) ::
```

Comment:
I'm very happy to see this since it makes this part of the public API for the first time (I think).  Date parts vary a lot amongst calendars, time parsing is commonly more uniform.